modules/ROOT/pages/cloud/overview/overview.adoc

```diff
@@ -0,0 +1,100 @@
+
+- Invoke all endpoints directly from your browser.
+- Inspect the autogenerated cURL request based on the input parameters you provide.
+- See the request and response schema for different HTTP status codes, and request and response examples. 
+
+Before you can issue requests from the browser against the API, make sure you follow these steps:
+
+. In the API reference, click *Authentication* in the sidebar.
+. Click the *Get Token* button. Do not edit the input field next to the button.
+. If successful, the text “1 API key applied” displays under the *Authentication* section of this page. The token is valid for an hour.
+. Choose the correct API server for your request.
+
+WARNING: Your API requests are executed against your actual environment and data, not a sandbox. 
+
+=== Make a request to the Control Plane API on the reference page
+
+For requests to the Control Plane API endpoints:
+
+. Click *API Servers* in the sidebar. 
+. Select “https://api.redpanda.com - Control Plane API”. 
+. From the Control Plane API endpoints reference, you can make a request by clicking the *Try* button. Make sure to enter any required parameter values, and provide the request body object if necessary.
+
+=== Make a request to the Data Plane APIs on the reference page
+
+For requests to the Data Plane API endpoints: 
+
+. Make a Get Cluster request for your target cluster.
+. The Get Cluster response contains the Data Plane API URL. Copy the value of `dataplane_api.url` from the response body. 
+. Click *API Servers* in the sidebar and select “\{dataplane_api_url} - Data Plane API”.
+. Paste the URL into the `dataplane_api_url` input field.
+. From the Data Plane API endpoints reference, you can make a request by clicking the *Try* button. Make sure to enter any required parameter values, and provide the request body object if necessary.
+
+== Response codes and errors
+
+If a request is successful, this is indicated by a `2xx` HTTP status code:
+
+- `200` OK. Request was successful.
+- `201` Created. The requested resource was successfully created.
+- `202` Accepted. The request is accepted and is processing, for example for a long-running operation.
+- `204` No content. Usually returned by a successful delete request.
+
+Errors are indicated by the following HTTP status codes:
+
+- `400` Bad request. There is an error that needs to be fixed in the request, such as an invalid parameter value. Attempt the request again after it is fixed.
+- `401` Unauthenticated. The authentication credentials are missing or invalid.
+- `404` Not found. The specified resource was not found.
+- `409` Conflict. Usually returned when the resource to be created by the request already exists.
+- `500` Internal server error. Reach out to Redpanda support.
+
+For errors, the response also includes a GRPC error code and details. See the https://connectrpc.com/docs/protocol/#error-codes[official GRPC documentation^] for further details.
```

Comment:
I do not have strong opinions on this... But the linked docs are actually not official gRPC docs, but for a slightly different framework that we use. 
The official gRPC docs are 
https://grpc.io/docs/guides/status-codes/
https://github.com/grpc/grpc/blob/master/doc/http-grpc-status-mapping.md
But I agree in some ways Connect docs are somewhat more relevant and useful and thorough. Also in some ways linking to to ConnectRPC leaks implementation details that may or may not be all that relevant or useful to the client / user. So not sure what the right approach is i guess...
- Reword so not to use "official gRPC" within the link.
- Link to actual gRPC documentation.
- Make our own table with examples perhaps. Essentially something that the ConnectRPC page does, but with specific examples of error responses from our API.

Maybe others have an opinion or want to chime in.

Reply:
@bojand Sorry for the delay - the link is my mistake, @sago2k8 had actually provided these links but I picked the Connect doc to include. This official doc https://grpc.io/docs/guides/status-codes/ appears a bit more helpful to me compared to the doc on GitHub... Am I on the right track? I'll switch out the link and if anyone else has any thoughts we can revisit.